src/Instrumentation/Slim/src/SlimInstrumentation.php

```diff
@@ -37,8 +37,8 @@ public static function register(): void
             RoutingMiddleware::class,
             'performRouting',
             pre: null,
-            post: static function (RoutingMiddleware $middleware, array $params, ServerRequestInterface $request, ?Throwable $exception) {
-                if ($exception) {
+            post: static function (RoutingMiddleware $middleware, array $params, ?ServerRequestInterface $request, ?Throwable $exception) {
```

Comment:
IMO, would be good to add comment about routing fail consequences

Reply:
done!